pylabrobot/liquid_handling/backends/hamilton/STAR.py

```diff
@@ -1225,6 +1225,7 @@ async def setup(self):
     left_x_drive_configuration_byte_1 = left_x_drive_configuration_byte_1[2:]
     self.core96_head_installed = left_x_drive_configuration_byte_1[2] == "1"
     self.iswap_installed = left_x_drive_configuration_byte_1[1] == "1"
+    self.autoload_installed = self.request_autoload_installed_after_2015()
```

Comment:
Should we print a warning if installed before 2015?

Reply:
Yes, I think this is a great idea.

I have chosen 2015 because it seemed like long enough ago but not too long... if it correctly identifies the presence/absence of an autoload we can make it anything we want.

Reply:
Whether autoload is installed should actually be available in the third bit of the `kb` parameter of the response to 'Request machine configuration' (configuration data 1). There is an example for the similar (I assume) left_x_drive_configuration_byte_1 in STAR.setup. Could you see if that works?

Reply:
Ohh nice; I will try this out first thing tomorrow. Thank you for the tip.

Reply:
This works like a charm. 
I have the deleted the intermediate "check download date" solution and it is now replaced with the more robust `STAR.request_machine_configuration()` check for bit 3.